initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -17,12 +17,13 @@
  <Software>
   <DropZone build="_"
             buildSet="DropZone"
-            computer="localhost"
             description="DropZone process"
             directory="${RUNTIME_PATH}/mydropzone"
             name="mydropzone"
             ECLWatchVisible="true"
-            umask="022"/>
+            umask="022">
+            <ServerList name="ServerList" server="localhost"/>
```

Comment:
It wouldn't make a lot of sense in multinode systems, but it does on 1 node system.
However, we tend to use ".", meaning 1st preferred NIC ip, which would avoid this issue.